<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<HTML>
 <HEAD>
   <TITLE> [mod_python] capturing and handling python errors in mod_python
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20capturing%20and%20handling%20python%20errors%20in%20mod_python&In-Reply-To=1132005143.25570%40dscpl.user.openhosting.com">
   <META NAME="robots" CONTENT="index,nofollow">
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="019536.html">
   <LINK REL="Next"  HREF="019537.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[mod_python] capturing and handling python errors in mod_python</H1>
<table border=0 width="100%"><tr><td valign="top">
    <B>Waitman Gobble</B> 
    <A HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20capturing%20and%20handling%20python%20errors%20in%20mod_python&In-Reply-To=1132005143.25570%40dscpl.user.openhosting.com"
       TITLE="[mod_python] capturing and handling python errors in mod_python">waitman at waitman.net
       </A><BR>
    <I>Mon Nov 14 16:59:39 EST 2005</I>
    <P><UL>
        <LI>Previous message: <A HREF="019536.html">[mod_python] capturing and handling python errors in mod_python
</A></li>
        <LI>Next message: <A HREF="019537.html">[mod_python] capturing and handling python errors in mod_python
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#19538">[ date ]</a>
              <a href="thread.html#19538">[ thread ]</a>
              <a href="subject.html#19538">[ subject ]</a>
              <a href="author.html#19538">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>
Hey Graham,

Thanks for the reply.

I think I figured out most of my issues. I had some mod_python config
directives in a global context in httpd.conf. The first time I tried to
get it going I put some stuff outside my VirtualHost directive. Apparently
this can cause wierd and unexpected, or maybe even normal and expected,
things to happen inside the VirtualHost config. In my case it was an
*oops*

I don't think that every request should go through the Logging thingy with
the (files exclusion) in my example. It looks like I created that
behaviour with the global defines. After the Global config stuff was
removed then it stopped logging the image, css requests in my log.py
script. at least it looks that way at the moment :-)



-- 
Waitman Gobble
<A HREF="http://waitman.net/">http://waitman.net/</A>
(707) 237-6921

</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="019536.html">[mod_python] capturing and handling python errors in mod_python
</A></li>
	<LI>Next message: <A HREF="019537.html">[mod_python] capturing and handling python errors in mod_python
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#19538">[ date ]</a>
              <a href="thread.html#19538">[ thread ]</a>
              <a href="subject.html#19538">[ subject ]</a>
              <a href="author.html#19538">[ author ]</a>
         </LI>
       </UL>

</td>
<td align="right" valign="top">
<script type="text/javascript"><!--
google_ad_client = "pub-9718360309690383";
google_ad_width = 120;
google_ad_height = 600;
google_ad_format = "120x600_as";
google_color_border = "CCCCCC";
google_color_bg = "FFFFFF";
google_color_link = "000000";
google_color_url = "666666";
google_color_text = "333333";
//--></script>
<script type="text/javascript"
  src="DISABLEhttp://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
</td>
</table>

<hr>
<a href="http://mailman.modpython.org/mailman/listinfo/mod_python">More information about the Mod_python
mailing list</a><br>
</body></html>
